src/js/components/main/board.js

```diff
@@ -0,0 +1,293 @@
+export const keyCodes = {
```

Comment:
I'd recommend to move this constant to a separate file. It's pretty big and you'll benefit a better board file code readability plus key config file that you can see in project structure and edit separately.

Reply:
Moved this constant to separete file